AppleStoreProject/AppleStoreProject/ForYouViewController.swift

```diff
@@ -7,4 +7,275 @@
 
 import UIKit
 
-final class ForYouViewController: UIViewController {}
+final class ForYouViewController: UIViewController {
+
+    //MARK: - Private Properties
+    private let forYouUnderLineView = UIView()
+    private let userImageButton = UIButton()
+    private let someThingNewLable = UILabel()
+    private let orderView = UIView()
+    private let orderLineView = UIView()
+    private let orderImage = UIImageView()
+    private let orderStatusImage = UIImageView()
+    private let orderStatusLable = UILabel()
+    private let orderStatusInformationLable = UILabel()
+    private let orderStatusButton = UIButton()
+    private let orderProgressView = UIProgressView()
+    private let orderProcessedLable = UILabel()
+    private let orderSendLable = UILabel()
+    private let orderDeliveredLable = UILabel()
+    private let recomendationLable = UILabel()
+    private let informationDeliveryImage = UIImageView()
+    private let informationDeliveryLable = UILabel()
+    private let notificationDeliveryLable = UILabel()
+    private let notificationDeliveryImage = UIImageView()
+    private let notificationDeliveryUnderLineView = UIView()
+    private let yourDevicesLable = UILabel()
+    private let showDevicesButton = UIButton()
+    private let mainScrollView = UIScrollView()
+
+    //MARK: - UIForYouViewController
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.title = "Для вас"
+        navigationController?.navigationBar.prefersLargeTitles = true
+
+        let barButton = UIBarButtonItem()
+        barButton.customView = userImageButton
+        navigationItem.rightBarButtonItem = barButton
+
+        mainScrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height)
+        view = mainScrollView
+
+        view.backgroundColor = .white
+
+        addVisualElements()
+        saveAvatarImage()
+    }
+
+    //MARK: - Methods
+    private func saveAvatarImage() {
+        guard let data = UserDefaults.standard.value(forKey: "avatar") as? Data else { return }
+        guard UIImage(data: data) != nil else { return }
```

Comment:
```suggestion
        guard let image = UIImage(data: data)  else { return }
```
смысл от этой конструкции , может так?